base_field_validator/ir_model_field_regex.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2014 Agile Business Group sagl (<http://www.agilebg.com>)
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as published
+#    by the Free Software Foundation, either version 3 of the License, or
+#    (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+from openerp import models, fields
+
+
+class IrModelFieldsRegex(models.Model):
+    _name = "ir.model.fields.regex"
+    name = fields.Char('Description', size=512, required=True)
```

Comment:
Can you remove size param?
In v8 [size don't is required](https://github.com/odoo/odoo/blob/282fc4a8505fba146b65cd484d35123b7de3bc96/openerp/fields.py#L1116)